### ISSUE TITLE
Adding version info to VertUI in order to tell what version is installed

### ISSUE THREAD

**Issue by luzpaz** [StudioPetrikas/FreeCADLink_VertUI#4]
For now, maybe in the source .qss file? But in the future it may be nice to have that info on the 'About' page. Perhaps opening a ticket on upstream [FreeCAD-Addons](https://github.com/FreeCAD/FreeCAD-addons/) asking for this capability?

**Comment by StudioPetrikas** [comment]
YES, that's a great idea. I was meaning to do some "patch-notes", so I can explain the choices I make. I've been meaning to do an overhaul in the near feature, so we can safely call this current version v1.0.

Regarding the info in the about section - let's keep that in mind for now. I'd like to be fully happy with the .qss first, we can then polish up the delivery.

**Comment by luzpaz** [comment]
Opened an upstream ticket nonetheless, since this has been a long-standing feature I think would make bug troubleshooting SO much easier for everyone (now that Add-ons are established and a legit thing in FC) 
https://github.com/FreeCAD/FreeCAD-addons/issues/199

**Comment by luzpaz** [comment]
Also added VertUI to https://github.com/FreeCAD/Addons/issues/16

**Comment by luzpaz** [comment]
Stylesheet versions will be shown in About info for 1.0 release
